sjb/config/common/test_cases/origin_release.yml

```diff
@@ -8,4 +8,6 @@ extensions:
       script: |-
         export OS_BUILD_IMAGE_ARGS=''
         hack/build-base-images.sh
-        make release
\ No newline at end of file
+        OS_BUILD_ENV_PRESERVE="_output/local" hack/env make build-rpms
+        hack/build-images.sh
```

Comment:
We decided not to use `hack/env make release` because it interacted poorly with this script, right?